subprojects/language-native/src/test/groovy/org/gradle/language/nativeplatform/internal/incremental/IncrementalCompileProcessorTest.groovy

```diff
@@ -516,6 +516,7 @@ class IncrementalCompileProcessorTest extends Specification {
 
     private HashCode getContentHash(File file) {
         virtualFileSystem.update([file.absolutePath], {})
-        return virtualFileSystem.readRegularFileContentHash(file.getAbsolutePath(), { it }).orElse(new MissingFileSnapshot(file.getAbsolutePath(), file.getName()).hash)
+        return virtualFileSystem.readRegularFileContentHash(file.getAbsolutePath(), { it })
+            .orElse(new MissingFileSnapshot(file.getAbsolutePath(), file.getName()).hash)
```

Comment:
This always creates a `MissingFileSnapshot`.

Reply:
Yes, though it is in a unit test, so I don't want to bother.

Reply:
Ah, not used to seeing Java 8 code in tests :)